### ISSUE TITLE
Multirun with PyTorch Ligtning (TensorboardLogger) ends up in prefixing logs

### ISSUE THREAD

**Issue by daandres** [allegroai/clearml#312]
Hello! 

Thanks for the great Project Dashboard. I'm just digging deeper in all features... 

I trained multiple models with different configuration for a custom hyperparameter search. I use pytorch_lightning and its logging (`TensorboardLogger`). When running my training script after `Task.init()` ClearML auto-creates a Task and connects the logger output to the server.

I log for each straining stage `train`, `val` and `test` the following scalars at each epoch: `loss`, `acc` and `iou`

When I have multiple configuration, e.g. `networkA` and `networkB` the first training log its values to loss, acc and iou, but the second to `networkB:loss`, `networkB:acc` and `networkB:iou`. This makes values umcomparable.

My training loop with Task initalization looks like this:

```
names = ['networkA', networkB']
for name in names:
     task = Task.init(project_name="NetworkProject", task_name=name)
     pl_train(name)
     task.close()
```
method `pl_train` is a wrapper for whole training with Pytorch Ligtning. No ClearML code is inside this method.

Do you have any hint, how to properly use the usage of a loop in a script using completly separated tasks?

---
I asked this on StackOverflow and I should opened a bug here. 
See this thread: https://stackoverflow.com/questions/66279581/clearml-multiple-tasks-in-single-script-changes-logghed-value-names/66285861

**Comment by bmartinn** [comment]
Thanks @daandres 
We were able to reproduce it and a fix is already committed to the main branch 🎊 
Feel free to test it (RC will be out soon)
```
pip install git+https://github.com/allegroai/clearml.git
```

**Comment by daandres** [comment]
Thanks for the fast fix. 
As I tested it, it seems to be fixed in master.
